test/TestHelper.cpp

```diff
@@ -890,4 +901,42 @@ void Listener::notifyTestFinished()
 		g_listener->testFinished();
 }
 
+size_t TestOutputHelper::m_currTest = 0;
```

Comment:
this do belong to a class. TestOutputHelper. 
a static class for handling output of the tests.